examples/BERT/shard_model.py

```diff
@@ -0,0 +1,30 @@
+import torch.nn as nn
+from model import TransformerEncoderLayer, XLMREmbedding
+
+class XLMRModelShards():
```

Comment:
`XLMRModelShards` and `MLMShards` literally are three utility functions to construct `XLMREmbedding`, `TransformerEncoder`, and `MLM`. Do we really need them here?

Reply:
These are just factories to create models' shards, they are not required but make code cleaner